include/open62541/client.h

```diff
@@ -280,6 +280,25 @@ UA_Client_connectUsername(UA_Client *client, const char *endpointUrl,
     return UA_Client_connect(client, endpointUrl);
 }
 
+/* Connect to the server and create+activate a Session with the given X.509 certificate.
+ * This first set the X509IdentityToken in the client config and
+ * then calls the regular connect method. */
+static UA_INLINE UA_StatusCode
+UA_Client_connectCertificate(UA_Client *client, const char *endpointUrl,
+                             const char *policyId, const char *certificateData) {
+    UA_X509IdentityToken* identityToken = UA_X509IdentityToken_new();
+    if(!identityToken)
+        return UA_STATUSCODE_BADOUTOFMEMORY;
+    identityToken->policyId = UA_STRING_ALLOC(policyId);
```

Comment:
UA_STRING_ALLOC could also fail due to out of memory. does this also need to be checked?